index.html

```diff
@@ -21,7 +21,34 @@ <h1>RustFest {{site.location.city}} {{site.dates.year}}</h1>
           {{site.dates.workshops}}<br />
         </p>
       </div>
-
+      <div></div>
+      <div class="location">
+	<br/>
```

Comment:
Are those breaks here required?

Reply:
I did this on purpose to make it look better the `<div></div>` block is a placeholder because we have 3 cell in a row, and the <br> is just to give a bit of spacing, probably can be done in a better way with a bit of css .... i'm not expert though

Reply:
¯\_(ツ)_/¯

Let's not block on this. we can fix this little stuff afterwards.